R/demo_fun.R

```diff
@@ -10,6 +10,7 @@
 #' @default The first letter of the alphabet
 #' @keywords internal
 #' @examplesx
+#' @info This is the introduction.
 #' @caption A simple example
 #' @info This is a simple example showing the default behaviour.
```

Comment:
two `@info` should one be `@intro`

Reply:
But functionally the intro is the same as an info so do we really need a new object?

Reply:
I wouldn't add an `@intro` tag. We would need to add an `@introcode` tag then too. As Edoardo pointed out the functionality would be the same. So it would just make the roclet code more complicated and the users would need to learn another two tags.

Reply:
okay sounds good!